cypress/e2e/types/constants.ts

```diff
@@ -212,3 +212,6 @@ export enum Languages {
     Go = "Go",
     Java = "Java",
 }
+
+export const itemsPerPage = 100;
```

Comment:
I don't think this needs to be a constant, this can be directly written in the code.

Reply:
Why not? I see it repeated in many test files

Reply:
Looks okay to make it a const.

Reply:
Agreed with Alex. Sometimes I need 10 when I am testing pagination... we can just put a value required in this or that particular case, agree? Don't see this to be usefull, to be honest

Reply:
To be honest, I just copy-pasted the `open` method from another model, could you please explain what is the use case of `selectItemsPerPage(itemsPerPage);` especially in the `open` method?

Reply:
This parameter means how many items should be shown on the page, `.open()` method is passing it to respective function that changes things on the page.

Reply:
> Why not? 

Because writing `itemsPerPage` is way longer that just writing `100`, and the fact that the constant is called `itemsPerPage` instead of just the value can cause confussion because just by tha name we don't know if `itemsPerPage` has a value of 100 or 10.

However I understand that it is a matter of preference in this case so we can keep the constant.